test.js

```diff
@@ -1,7 +1,7 @@
 /* eslint new-cap:0 no-new:0 */
 'use strict';
 
-var bufferToStream = require('simple-bufferstream');
+var sbuff = require('simple-bufferstream');
```

Comment:
No reason, rvagg required it under that name in his [docs](https://github.com/rvagg/node-simple-bufferstream). When I replaced the other module with this one in index.js, I used that name. Later I changed the tests for consistency.

Would you like me to change it back?